Dockerfile

```diff
@@ -9,6 +9,9 @@ COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
 
+# install dependencies for text extraction (refer https://github.com/sajari/docconv)
+RUN apt-get install poppler-utils wv unrtf tidy
```

Comment:
We are using multi-stage docker build. I guess adding here is not enough?

Reply:
updated to use another base image